main/webapp/modules/core/scripts/index.js

```diff
@@ -68,28 +68,31 @@ Refine.selectActionArea = function(id) {
 };
 
 $(function() {
-  var isThereNewRelease = function() {
-    // TODO: This eeds to be modified to do checking based solely on version, not revision
-    var thisRevision = OpenRefineVersion.revision;
-    var thisVersion = OpenRefineVersion.version;
-
-    var revision_pattern = /r([0-9]+)/;
-
-    if (!revision_pattern.test(thisRevision)) { // probably "trunk"
+  var isThereNewRelease = function(thisVer,latestVer) {
+    // Assumes Semantic Version numbering format of form X.Y.Z where X, Y, and Z are non-negative integer
+    // Ignores any trailing letters e.g. 2.7.1-rc.1 will be interpreted as equivalent to 2.7.1
+    if(thisVer == latestVer) {
       return false;
     }
-
-    var latestRevision = OpenRefineReleases.releases[0].revision;
-    var latestVersion = OpenRefineReleases.releases[0].version;
-
-    var thisRev = parseInt(revision_pattern.exec(thisRevision)[1],10);
-    var latestRev = parseInt(revision_pattern.exec(OpenRefineReleases.releases[0].revision)[1],10);
-
-    // Parser version into main version and suffix which follows dash
-    // Parse version from dot separated string into array of integers
-    
-    // compare left to right, including suffix
-    return latestRev > thisRev;
+    var thisVerParts = thisVer.split(".");
+    var latestVerParts = latestVer.split(".");
+    while(thisVerParts.length < 3)
+      thisVerParts.push(0);
+    while(latestVerParts.length < 3)
+      latestVerParts.push(0);
+
+    for(var i=0; i<3; i++) {
+      var thisVerPartInt = parseInt(thisVerParts[i],10);
+      var latestVerPartInt = parseInt(latestVerParts[i],10);
+      if(thisVerPartInt == latestVerPartInt) {
+        continue;
+      } else if (thisVerPartInt > latestVerPartInt) {
+        return false;
+      } else {
+        return true;
+      }
```

Comment:
Things of the form `if foo then return false else return true` can be rewritten as `return not foo`.

Reply:
Thanks - will re-write that.
In terms of opening link in new window, I didn't change the notification code, so it is just following its previous behaviour. I think opening in a new window makes sense - I'll amend